base/log_level.go

```diff
@@ -82,3 +82,8 @@ func (l *LogLevel) UnmarshalText(text []byte) error {
 	}
 	return fmt.Errorf("unrecognized log level: %q (valid options: %v)", string(text), logLevelNames)
 }
+
+// logLevelPtr is a convenience function that returns a pointer to the given logLevel
+func logLevelPtr(logLevel LogLevel) *LogLevel {
+	return &logLevel
+}
```

Comment:
Is this used/needed?

Reply:
Whoops, this is a leftover from a previous version of the unit tests. I'll go ahead and remove.